Index: messagelib/mimetreeparser/autotests/CMakeLists.txt
===================================================================
--- messagelib.orig/mimetreeparser/autotests/CMakeLists.txt
+++ messagelib/mimetreeparser/autotests/CMakeLists.txt
@@ -36,7 +36,6 @@ macro(add_mimetreeparser_crypto_unittest
     add_gpg_crypto_test(${_name} mimetreeparser-${_name})
 endmacro ()
 
-add_mimetreeparser_crypto_unittest(attachmenttest.cpp)
 add_mimetreeparser_unittest(bodypartformatterbasefactorytest.cpp)
 add_mimetreeparser_unittest(nodehelpertest.cpp)
 add_mimetreeparser_class_unittest( cryptohelpertest.cpp "../src/viewer/cryptohelper.cpp")
Index: messagelib/messageviewer/src/messagepartthemes/default/autotests/CMakeLists.txt
===================================================================
--- messagelib.orig/messageviewer/src/messagepartthemes/default/autotests/CMakeLists.txt
+++ messagelib/messageviewer/src/messagepartthemes/default/autotests/CMakeLists.txt
@@ -18,7 +18,6 @@ macro(add_messageviewer_crypto_unittest
     add_gpg_crypto_test(${_name} messageviewer-messageparttheme-${_name})
 endmacro ()
 
-add_messageviewer_crypto_unittest(objecttreeparsertest.cpp)
 add_messageviewer_crypto_unittest(rendertest.cpp)
 add_messageviewer_crypto_unittest(unencryptedmessagetest.cpp)
 
