From 3f9d16c7dadd2c98b00c5e7216cd69cfb518cab9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sandro=20Knau=C3=9F?= <sknauss@kde.org>
Date: Mon, 25 Jul 2016 12:39:15 +0200
Subject: [PATCH] make it impossible to override css settings from a html mail.

---
 messageviewer/src/viewer/csshelperbase.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/messageviewer/src/viewer/csshelperbase.cpp b/messageviewer/src/viewer/csshelperbase.cpp
index 8287e59..8825851 100644
--- a/messageviewer/src/viewer/csshelperbase.cpp
+++ b/messageviewer/src/viewer/csshelperbase.cpp
@@ -276,6 +276,9 @@ QString CSSHelperBase::printCssDefinitions(bool fixed) const
                        "div.fancy.header > div a[href] { color: %3 ! important; }\n\n"
 
                        "div.fancy.header > table.outer{\n"
+                       "  all: inherit;\n"
+                       "  width: auto ! important;\n"
+                       "  border-spacing: 0;\n"
                        "  background-color: %2 ! important;\n"
                        "  color: %3 ! important;\n"
                        "  border-bottom: solid %3 1px ! important;\n"
@@ -564,6 +567,9 @@ QString CSSHelperBase::screenCssDefinitions(const CSSHelperBase *helper, bool fi
                        "}\n\n"
 
                        "div.fancy.header > table.outer {\n"
+                       "  all: inherit;\n"
+                       "  width: auto ! important;\n"
+                       "  border-spacing: 0;\n"
                        "  background-color: %5 ! important;\n"
                        "  color: %4 ! important;\n"
                        "  border-bottom: solid %4 1px ! important;\n"
-- 
2.8.1

