--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,7 +71,8 @@ set(GRANTLEETHEME_LIB_VERSION "5.2.0")
 set(KDEPIM_APPS_LIBS_LIB_VERSION "5.2.0")
 set(LIBKLEO_LIB_VERSION "5.2.0")
 set(PIMCOMMON_LIB_VERSION "5.2.0")
-find_package(Gpgme REQUIRED)
+find_package(Gpgmepp "1.8.0" REQUIRED)
+find_package(QGpgme "1.8.0" REQUIRED)
 find_package(KF5KaddressbookGrantlee ${KDEPIM_APPS_LIBS_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5GrantleeTheme ${GRANTLEETHEME_LIB_VERSION} CONFIG REQUIRED)
 find_package(KF5Gravatar ${GRAVATAR_LIB_VERSION} CONFIG REQUIRED)
--- a/messagecomposer/autotests/CMakeLists.txt
+++ b/messagecomposer/autotests/CMakeLists.txt
@@ -28,7 +28,7 @@ macro( add_messagecomposer_cryptotest _s
         KF5::PimTextEdit
         KF5::AkonadiCore
         Qt5::Test
-        KF5::QGpgme
+        QGpgme
         KF5::IconThemes
         KF5::Completion
         KF5::MailTransport
--- a/messageviewer/autotests/CMakeLists.txt
+++ b/messageviewer/autotests/CMakeLists.txt
@@ -11,7 +11,7 @@ macro(add_messageviewer_unittest _source
     ecm_add_test(${_source} util.cpp setupenv.cpp testcsshelper.cpp ../src/viewer/cryptohelper.cpp
         TEST_NAME ${_name}
         NAME_PREFIX "messageviewer-"
-        LINK_LIBRARIES KF5::MessageViewer KF5::Libkleo KF5::QGpgme Qt5::Test Qt5::WebKitWidgets KF5::KIOCore KF5::Mime KF5::AkonadiCore KF5::AkonadiNotes KF5::CalendarCore
+        LINK_LIBRARIES KF5::MessageViewer KF5::Libkleo QGpgme Qt5::Test Qt5::WebKitWidgets KF5::KIOCore KF5::Mime KF5::AkonadiCore KF5::AkonadiNotes KF5::CalendarCore
     )
 endmacro ()
 
@@ -32,7 +32,7 @@ macro(add_messageviewer_crypto_unittest
     target_link_libraries( ${_name}
         KF5::MessageViewer
         KF5::Libkleo
-        KF5::QGpgme
+        QGpgme
         Qt5::Test
         Qt5::WebKitWidgets
         KF5::KIOCore
--- a/messageviewer/src/CMakeLists.txt
+++ b/messageviewer/src/CMakeLists.txt
@@ -340,8 +340,6 @@ target_link_libraries(KF5MessageViewer
     )
 if (WIN32)
     target_link_libraries(KF5MessageViewer PUBLIC KF5::Gpgmepp)
-else()
-    target_link_libraries(KF5MessageViewer PUBLIC KF5::Gpgmepp-pthread)
 endif()
 
 target_include_directories(KF5MessageViewer PUBLIC $<TARGET_PROPERTY:KF5::WebKit,INTERFACE_INCLUDE_DIRECTORIES>)
--- a/templateparser/autotests/CMakeLists.txt
+++ b/templateparser/autotests/CMakeLists.txt
@@ -15,7 +15,7 @@ macro(add_templateparser_unittest _sourc
     KF5::Libkleo
     KF5::TemplateParser
     KF5::MessageViewer
-    KF5::QGpgme
+    QGpgme
     Qt5::Test
     Qt5::WebKitWidgets
     KF5::KIOCore
